.github/workflows/deploy.yml

```diff
@@ -0,0 +1,63 @@
+name: Deploy Testnet
+
+permissions:
+  contents: write
+  pull-requests: read
+
+env:
+  VERSION: ${{ github.event.inputs.tag }}
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: What is the release tag? (e.g. "1.0.2")
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check release version
+        id: release-version
+        env:
+          VERSION: ${{ env.VERSION }}
+        run: |
+          if [[ ! "${VERSION}" =~ ^[0-9]+\.[0-9]+\.[0-9]+.*$ ]]; then
+              echo "Invalid version tag '${VERSION}'"
+              exit 1
+          fi
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: SSH into Compute Engine
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.VPS_HOST }}
+          username: ${{ secrets.VPS_USERNAME }}
+          key: ${{ secrets.VPS_PRIVATE_KEY }}
+          port: 22
+          script: |
+            cd ${{ secrets.PROJECT_DIRECTORY }}
+            git switch main
+            git pull origin main
+            export NVM_DIR=~/.nvm
+            source ~/.nvm/nvm.sh
+            pnpm install
+            pnpm run prod
+
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@v4.1.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Isn't this going to overwrite/be overwritten by the `generate_release_notes` option from the `Release` job ?

Reply:
According to the [document](https://github.com/softprops/action-gh-release) about `generate_release_notes` says:

> Whether to automatically generate the name and body for this release. If name is specified, the specified name will be used; otherwise, a name will be automatically generated. If body is specified, the body will be pre-pended to the automatically generated notes. See the [GitHub docs for this feature](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) for more information